aws-s3-public-bucket/main.tf

```diff
@@ -0,0 +1,82 @@
+locals {
+  tags = {
+    project         = var.project
+    env             = var.env
+    service         = var.service
+    owner           = var.owner
+    managedBy       = "terraform"
+    isPublic        = true
+    public_reason   = var.public_reason
+    bucket_contents = var.bucket_contents
+  }
+
+  // make sure we have public in the name
+  bucket_name = replace(var.bucket_name, "public", "") == var.bucket_name ? "${var.bucket_name}-public" : var.bucket_name
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>local.bucket_name is declared but not used